cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -419,6 +419,14 @@
     </description>
   </property>
 
+  <property>
+    <name>upgrade.timeout.secs</name>
```

Comment:
better call `app.meta.upgrade.timeout.sec`? since its not the upgrade timeout but part of it.